blog/migrations/__init__.py

```diff
@@ -0,0 +1 @@
+"""Django website."""
```

Comment:
No, it'd DB migration, not a website.

Reply:
I just have to delete? No need to write here?

Reply:
Just write that it's migrations. Because the current docstring is a lie.